src/app.cpp

```diff
@@ -347,8 +349,8 @@ uint64_t calculate_startdate(Vector::BLF::File* infile) {
 }
 
 int main(int argc, char* argv[]) {
-	if (argc != 3) {
-		fprintf(stderr, "Usage %s [infile] [outfile]\n", argv[0]);
+	if (argc != 4) {
+		fprintf(stderr, "Usage %s [infile] [outfile] [mappingfile]\n", argv[0]);
```

Comment:
Use same command line syntax as TECMP converter (you can copy the command line parsing code)